rules/merge_inputs.smk

```diff
@@ -1,4 +1,31 @@
-# ------------- helper functions to collect, merge & download input files ------------------- #
+"""
+This part of the workflow merges inputs based on what is defined in the config.
+
+OUTPUTS:
+
+    metadata  = results/metadata.tsv
+    sequences = results/sequences_{segment}.fasta
+
+The config dict is expected to have a top-level `inputs` list that defines the
+separate inputs' name, metadata, and sequences. Optionally, the config can have
+a top-level `additional-inputs` list that is used to define additional data that
+are combined with the default inputs:
+
+```yaml
+inputs:
+    - name: default
+      metadata: <path-or-url>
+      sequences: <path-or-url>
```

Comment:
sequences can be a dictionary as well (and ditto for `additional_inputs`)

Reply:
Thanks for flagging. Added to the docstring in [force-push](https://github.com/nextstrain/avian-flu/compare/426f8df9547e15018cd7933ffb40f52489e7cb12..fc4a6f4e8914a56e9c9b3d3cc57fc5f76863cbec)